src/Hosting/Hosting/src/Internal/HostingApplicationDiagnostics.cs

```diff
@@ -172,7 +172,7 @@ private void LogRequestStarting(HttpContext httpContext)
             // IsEnabled is checked in the caller, so if we are here just log
             _logger.Log(
                 logLevel: LogLevel.Information,
-                eventId: LoggerEventIds.RequestStarting,
+                eventId: new EventId(LoggerEventIds.RequestStarting, nameof(LoggerEventIds.RequestStarting)),
```

Comment:
I would avoid `nameof` here. Even though we don't intent to change this, if we did ever change the name of the property it would break consumers. We started using `nameof` originally but decided it was better to just use string constants. The goal of `nameof` is to ensure that when you rename the target of the `nameof` expression, your reference is updated which is actually *not* what we want here.

Reply:
I hadn't thought about it that way at first, but I can see your point.  This should be corrected in the latest commit.